env/base.env

```diff
@@ -0,0 +1,4 @@
+LD_PROJECT_TYPE="base"
+FRONTEND_REPOSITORY=${FRONTEND_REPOSITORY:-https://github.com/drunomics/lupus-decoupled-nuxt3-demo}
```

Comment:
changing frontend repository defaults depending on recipe is confusing imo. let's not do this. when it should be changed, the links can set the right frontend repo also